plugins/bundle/plugin.go

```diff
@@ -445,7 +445,14 @@ func (p *Plugin) activate(ctx context.Context, name string, b *bundle.Bundle) er
 
 		// Compile the bundle modules with a new compiler and set it on the
 		// transaction params for use by onCommit hooks.
-		compiler := ast.NewCompiler().WithPathConflictsCheck(storage.NonEmpty(ctx, p.manager.Store, txn))
+		// If activating a delta bundle, use the manager's compiler which should have
+		// the polices compiled on it.
+		var compiler *ast.Compiler
+		if b.Manifest.Type == bundle.DeltaBundleType {
+			compiler = p.manager.GetCompiler()
```

Comment:
Is it possible for p.manager.GetCompiler() to return nil? What would happen? Should we lazily create a compiler in that case?